src/display/annotation_layer.js

```diff
@@ -976,25 +976,29 @@ class CheckboxWidgetAnnotationElement extends WidgetAnnotationElement {
     const element = document.createElement("input");
     element.disabled = data.readOnly;
     element.type = "checkbox";
-    element.name = this.data.fieldName;
+    element.name = data.fieldName;
     if (value) {
       element.setAttribute("checked", true);
     }
     element.setAttribute("id", id);
+    element.setAttribute("acroformExportValue", data.exportValue);
```

Comment:
The attribute name feels a bit verbose to me; any reason that just `exportValue` wouldn't work?